src/python/scripts/run_gerp.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Calculates the Genomic Evolutionary Rate Profiling (GERP) of a multiple sequence alignment.
+
+GERP will produce two files, one with the conservation scores (suffixed with ".rates") and another with
+the constrained elements (suffixed with ".elems").
+
+Typical usage example::
+
+    $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw
+
+If ``gerpcol`` and/or ``gerpelem`` are not accesible from ``$PATH``, you will need to provide the
+path where they can be found::
+
+    $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw --gerp_exe_dir path/to/gerp/
+
+"""
+
```

Comment:
This will be needed if the suggestions below are accepted.
```suggestion

import os
```